scripts/sync_project.rb

```diff
@@ -59,13 +59,15 @@ def sync_firestore()
   s.test_groups = [
```

Comment:
Could you remove the line above excluding `serializer_test.cc` (and the `b/79496027` comment) and then re-run `sync-project.rb`?

Reply:
Also there's a line in `scripts/check_test_inclusion.py` tagged with that bug number too that needs cleaning up.